gensim/models/word2vec.py

```diff
@@ -345,6 +345,9 @@ def __init__(
             :meth:`~gensim.models.word2vec.Word2Vec.get_latest_training_loss`.
         callbacks : iterable of :class:`~gensim.models.callbacks.CallbackAny2Vec`, optional
             Sequence of callbacks to be executed at specific stages during training.
+        reduced_windows : bool, optional
+            If True, the window size is uniformly sampled from {1, `window`}
+            during training. Otherwise, it is fixed to `window`.
```

Comment:
I would extend this comment with more explanation, eg:

    If True, the effective window size is uniformly sampled from  [1, `window`] 
    for each target word during training, to match the original word2vec algorithm's 
    approximate weighting of context words by distance. Otherwise, the effective 
    window size is  always fixed to `window`.

Reply:
Okay, we will use this suggested docstring.